# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/printing/nsPrintData.h
# Commit: f6e9ace877bd
# Full Hash: f6e9ace877bde122f61baa5d298aea48ad26f8b6
# Author: Jonathan Watt <jwatt@jwatt.org>
# Date: 2022-05-23 09:52:44
# Regressor Bug: 1770539
# File Overlap Count: 1
# Description:
#   Bug 1770539 p1 - Move nsPrintData::mPrintSettings to nsPrintJob. r=emilio
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D146973
# ==============================================================================

diff -r 28d8297085fe -r f6e9ace877bd layout/printing/nsPrintData.h
--- a/layout/printing/nsPrintData.h	Wed May 18 03:37:27 2022 +0000
+++ b/layout/printing/nsPrintData.h	Sun May 22 09:37:55 2022 +0000
@@ -67,8 +67,6 @@
   int32_t mNumPrintablePages;
   float mShrinkRatio;
 
-  nsCOMPtr<nsIPrintSettings> mPrintSettings;
-
  private:
   nsPrintData() = delete;
   nsPrintData& operator=(const nsPrintData& aOther) = delete;